index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />    <link rel="stylesheet" href="style.css">
+    <title>Cars-connect-254 conference website</title>
+</head>
+<body>
+    <div class=" wrapper">
+            <nav class="topper">
+                <ul class="top-nav">
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li><a href="#"></a>English</li>
+                    <li><a href="#"></a>MyPage</li>
+                    <li><a href="#"></a>Logout</li>
+                </ul>
+            </nav>
+            <nav class="main-navbar">
+                
+                <div class="hamburger-menu">
+                    <div id="loggo">
+                        <a href="index.html"><img src="images/logo.png" alt="logo" id="logo"></a>
+                    </div>
+                    
+                    <input id="menu__toggle" type="checkbox" />
+                    <label class="menu__btn" for="menu__toggle">
+                        <span></span>
+                    </label>
+                    
+                        
+                    
+                        <ul class="menu__box" >
+                            <li><a class="menu__item" href="about.html">About</a></li>
+                            <li><a class="menu__item" href="#">Program</a></li>
+                            <li><a class="menu__item" href="#">Sponser</a></li>
+                            <li><a class="menu__item" href="#">Join</a></li>
+                            <li><a class="menu__item" href="#">News</a></li>
+                            <li id="campaign"><a class="menu__item" href="#">Campaign</a></li>
+                        </ul>
+
+                    </div>
+                    
+                </div>
+
```

Comment:
- [ ] As requested by the previous code reviewer  the menu items on the first part and the second part on your header is not properly aligned on the right as we have it on the design template. Take a look at the below screenshots and walk your design alongside the guiding template.

- [ ] Also kindly make sure the margin, the padding, the spacing are respected because you should stick to the design , that means , make sure the margin, the padding , the color of your design should look as much as possible as in the template design 
![Capture](https://user-images.githubusercontent.com/80458199/226098927-bad35518-a8ae-4ab9-81f1-14a235551529.PNG)